backend/server/controllers/courseinstances.js

```diff
@@ -521,7 +526,7 @@ module.exports = {
         if (req.body.issuesDisabled && isAllowedToUpdate === 'student') {
           return res.status(403).send('You cannot modify this flag as a student.')
         }
-        if (req.body.github && isAllowedToUpdate.includes('assistant')) {
+        if (req.body.github && isAllowedToUpdate === 'assistant') {
```

Comment:
This needs to be `.includes` since the return value can be "other assistant"

Reply:
Oh, don't know how it got there anyway since I didn't really make that change myself. I'll correct it now.